NebulaWorld/SimulatedWorld.cs

```diff
@@ -123,7 +123,15 @@ public static void UpdatePlayerColor(ushort playerId, Float3 color)
 
         public static void OnPlaceEntity(EntityPlaced packet)
         {
-            EntityManager.PlaceEntity(packet);
+            if (packet.isPrebuild && packet.planetId == GameMain.localPlanet?.id)
+            {
+                EntityManager.PlaceEntityPrebuild(packet);
+            }
+            else
+            {
+                // if this player is currently not on the planet where the building is placed then dont spawn a prebuild
+                EntityManager.PlaceEntity(packet);
```

Comment:
Wouldn't this place 2 of the same entity if the player is not on the same local planet? One for the prebuild and one for the real building?